README.md

```diff
@@ -5,13 +5,13 @@
 
 # Javalin - A simple web framework for Java and Kotlin
 
-Javalin is a very lightweight web framework for Kotlin and Java which supports WebSockets, HTTP2 and async requests. Javalin’s main goals are simplicity, a great developer experience, and first class interoperability between Kotlin and Java.
+Javalin is a very lightweight web framework for Kotlin and Java which supports WebSockets, HTTP2 and async requests. Javalin’s main goals are simplicity, a great developer experience and first class interoperability between Kotlin and Java.
```

Comment:
Oxford comma is a thing.  Not sure why we should remove it.

Reply:
It's indeed a thing, but I think the sentence flows better without it in this case.